orttraining/orttraining/python/ort_trainer.py

```diff
@@ -535,7 +535,7 @@ def __init__(self, model, loss_fn, model_desc, training_optimizer_name, map_opti
                  global_step=0, get_lr_this_step=None, loss_scaler=None, deepspeed_zero_stage=0,
                  enable_grad_norm_clip=True, frozen_weights=[], _opset_version=DEFAULT_OPSET_VERSION,
                  _enable_internal_postprocess=True, _extra_postprocess=None, _use_deterministic_compute=False,
-                 use_invertible_layernorm_grad=False):
+                 use_invertible_layernorm_grad=False, _debug_conversion=False):
```

Comment:
"conversion" is not very clear for user of what is being converted :)

`_debug_onnx_model_export` or `_debug_onnx_model_conversion`